docs/decisions/0009-course-catalog-info-changed-design.rst

```diff
@@ -11,19 +11,23 @@ Context
 
 The COURSE_CATALOG_INFO_CHANGED signal is used by Studio to convey that information relevant to the course catalog has changed. It was created with an eye towards eventually replacing the refresh-course-metadata batch job, which syncs all data between edx-platform and course-discovery.
 
-Refresh-course-metadata functions differently depending on whether or not the system has Publisher enabled. Specifically, after requesting information from the ``/courses`` endpoint in Studio, Discovery will ignore certain fields if Publisher is enabled. These fields are mostly part of the ``media`` attribute of the API response. They are sent in a variety of ways (absolute urls, paths, sometimes divided by size, etc.)
+Refresh-course-metadata functions differently depending on whether or not the system has Publisher enabled. Specifically, after requesting information from the ``/courses`` endpoint in Studio, Discovery will ignore certain fields if Publisher is enabled. Most, thought not all, of these fields are sent as part of the ``media`` attribute of the API response. These media fields are sent in a variety of ways (absolute urls, paths, sometimes divided by size, etc.)
 
 Decision
 --------
 
-The COURSE_CATALOG_INFO_CHANGED will only contain the information necessary to work in a Publisher-enabled environment. In particular, this means it will not contain the ``media`` field usually present in the Studio ``/courses`` API endpoint.
+The COURSE_CATALOG_INFO_CHANGED will only contain the information necessary to work in a Publisher-enabled environment. In particular, this means it will not contain the following fields usually present in the Studio ``/courses`` API endpoint:
+- ``media``
+- ``short_description``
+- ``mobile_available``
+- ``effort``
 
 In Discovery, if Publisher is not enabled, the consumer will log a warning and not try to update anything.
 
 Rationale
 ---------
 
-The way we update media information in refresh-course-metadata is quite haphazard, with no real standard way of sending over the information or of storing it on the Discovery end. Replicating these structures in the COURSE_CATALOG_INFO_CHANGED signal would make the data definition confusing.
+The way we update media information in refresh-course-metadata is quite haphazard, with no real standard way of sending over the information or of storing it on the Discovery end. Replicating these structures in the COURSE_CATALOG_INFO_CHANGED signal would make the data definition confusing. In addition, it is very difficult to test these other fields in the edX.org staging and production environments because, since publisher is enabled, these fields are not present in studio.
 
 Until these fields are needed by a consumer in a non-Publisher environment (if ever), it makes sense to defer trying to find a good solution to this problem.
```

Comment:
I would move the new sentence from above ("In addition, it is very difficult...") to this section. I think this sentence adds better context. Also, I'd try to avoid edx.org references if we can. Something like:
```suggestion
Until these fields are needed by a consumer in a non-Publisher environment (if ever), it makes sense to defer trying to find a good solution to this problem. Additionally, it is very difficult to test this without a real non-Publisher environment that is running refresh-course-metadata.
```

Reply:
Fixed (locally, not with commit suggestion)